src/ngraph/op/constant.cpp

```diff
@@ -78,6 +78,7 @@ string op::Constant::convert_value_to_string(size_t index) const
     case element::Type_t::u16: rc = to_string(get_vector<uint16_t>()[index]); break;
     case element::Type_t::u32: rc = to_string(get_vector<uint32_t>()[index]); break;
     case element::Type_t::u64: rc = to_string(get_vector<uint64_t>()[index]); break;
+    case element::Type_t::u1: throw runtime_error("unsupported type");
```

Comment:
u1 should be supported in constant

Reply:
Added convert_value_to_string support for u1.